Docker.md

```diff
@@ -0,0 +1,27 @@
+Frank!Framework with Docker
+===========================
+
+Docker images are provided, suitable both for local and server use. Images are provided from the registry located at <https://TBD>, the source is avalaible from the [docker-folder](docker/appserver/Tomcat) in this repository.
+
+General use
+===========
+
+The container contains the following important directories:
+| location | description |
+|---|---|
+| /opt/frank/testtool | For Larva tests that are included in the image |
+| /opt/frank/testtool-ext | For Larva tests that are mounted from the environment |
+| /usr/local/tomcat/lib/ | Contains drivers and other dependencies |
+
+The container also contains the following important files:
+| location | description |
+|---|---|
+| /usr/local/tomcat/conf/Catalina/localhost/iaf.xml | mount/copy of your context.xml |
```

Comment:
Misschien is het goed om een opmerking te maken dat 'localhost' in context.xml vervangen moet worden door 'host.docker.internal' (of dat je een truuk verzint waardoor dat automatisch gaat wanneer nodig)

Reply:
Ik heb er een opmerking bij gezet dat host.docker.internal gebruikt kan worden om terug te komen op de host-machine. 

Automatische bepaling gaat denk ik meer problemen opleveren dan dat het oplost, omdat context.xml echt omgeving-specifiek is en dus door de omgeving correct aangeleverd moet worden. Daarin gaan rommelen zal processen vereisen die voor Tomcat draaien en dan Tomcat starten, maar dat is flink wat extra complexiteit omdat je primaire proces in de container niet langer Tomcat is, maar dat script.